static/js/app.ts

```diff
@@ -218,7 +220,33 @@ export function runit(level: string, lang: string, cb: () => void) {
       }
       if (response.Error) {
         error.show(ErrorMessages['Transpile_error'], response.Error);
-        if (response.Location && response.Location[0] != "?") {
+        if (response.Invalid_command)  {
+          range = editor.find(response.Invalid_command, {
```

Comment:
A replacement for the `var range` above:

```suggestion
          const range = editor.find(response.Invalid_command, {
```